demo/rails/.env

```diff
@@ -1,3 +1,3 @@
-# these will only work on localhost (registered on mg@z) ... make your own at https://www.google.com/recaptcha
-RECAPTCHA_PUBLIC_KEY=6Le7oRETAAAAAETt105rjswZ15EuVJiF7BxPROkY
-RECAPTCHA_PRIVATE_KEY=6Le7oRETAAAAAL5a8yOmEdmDi3b2pH7mq5iH1bYK
+# reCAPTCHA keys for testing purposes (https://developers.google.com/recaptcha/docs/faq)
+RECAPTCHA_SITE_KEY = 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
```

Comment:
Alright. The reason I changed the keys was because the old ones were not working at all. And those are "official" ones meant for testing purposes.